common/djangoapps/util/password_policy_validators.py

```diff
@@ -1,322 +1,467 @@
-    descs = []
-    complexity = password_complexity()
-    if which is None:
-        which = complexity.keys()
-
-    for key in _allowed_password_complexity:  # we iterate over allowed keys so that we get the order right
-        value = complexity.get(key, 0) if key in which else 0
-        if not value:
-            continue
-
-        if key == 'ALPHABETIC':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} letter', '{num} letters', value).format(num=value))
-        elif key == 'UPPER':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} uppercase letter', '{num} uppercase letters', value).format(num=value))
-        elif key == 'LOWER':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} lowercase letter', '{num} lowercase letters', value).format(num=value))
-        elif key == 'DIGITS':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} digit', '{num} digits', value).format(num=value))
-        elif key == 'NUMERIC':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} number', '{num} numbers', value).format(num=value))
-        elif key == 'PUNCTUATION':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} punctuation mark', '{num} punctuation marks', value).format(num=value))
-        elif key == 'NON ASCII':  # note that our definition of non-ascii is non-letter, non-digit, non-punctuation
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} symbol', '{num} symbols', value).format(num=value))
-        elif key == 'WORDS':
-            # Translators: This appears in a list of password requirements
-            descs.append(ungettext('{num} word', '{num} words', value).format(num=value))
-        else:
-            raise Exception('Unexpected complexity value {}'.format(key))
 
-    return descs
+class UserAttributeSimilarityValidator(DjangoUserAttributeSimilarityValidator):
+    def get_instruction_text(self):
+        return ''
+
+    def get_restriction(self):
+        """
+        Returns a key, value pair for the restrictions related to the Validator
+        """
+        return None, None
 
 
-def password_instructions():
+class MaximumLengthValidator(object):
     """
-    :return: A string suitable for display to the user to tell them what password to enter
+    Validate whether the password is shorter than a maximum length.
+
+    Parameters:
+        max_length (int): the maximum number of characters to require in the password.
     """
-    min_length = password_min_length()
-    reqs = _password_complexity_descriptions()
+    def __init__(self, max_length=100):
```

Comment:
Is that a change of default? I thought our old code (and config) defaults were 75.